### PR TITLE
First JMAP MDN specification draft

### PR DESCRIPTION
**Pull request by rouazana**
This draft has also been submitted to IETF process. Please tell me if anything is wrong concerning this process.

Regards,
Raphaël Ouazana.

### CODE REVIEW

File: spec/mdn/mdn.mdown [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+- **accountId**: `String|null`
+  The id of the account to use for this call. If `null`, defaults to the `urn:ietf:params:jmap:mail` primary account.
+- **mdns**: `String[MDN]`
+  A map of creation id (client specified) to MDN objects
+
+An **MDN** object has the following properties:
+
+- **referencedMessageId**: `String`
+  Message Id of the received message the user wants to create an MDN for.
```
[the remark this answers is not in the document]
Comment (chibenwa) [reply spec/mdn/mdn.mdown]: Shouldn't we specify that this should be used for generating the Original-Recipient, Final-Recipient and Original-Message-ID fields?

**Comment by chibenwa** [comment]
Also **MessageSubmission** has a **mdnBlobIds** field.

Shouldn't this spec add a `mdns` field exposing parsed MDNs for the client?

**Comment by chibenwa** [comment]
https://tools.ietf.org/html/rfc3503 defines the `$MDNSent` keyword. I would suggest to at least reference this in this spec, and explicitly explain the role of this keyword.

**Comment by chibenwa** [comment]
In https://jmap.io/spec-mail.html#email-submission I propose a convenient property for setting the `Disposition-Notification-To` header to the right value when sending an email via JMAP.

My proposal would be to add a `dispositonNotificationTo` and a `dispositionNotificationOptions` to the `EmailSubmission` object. We should make it clear which effect it will have. We should make also clear which headers it sets.

That way we make it easier to request an MDN, without requiring JMAP users to read RFC-3798.

**Comment by neilj** [comment]
Hi guys. I think discussion about this draft would best be done on the JMAP mailing list. Send a message with a link to the draft you've published and a request for review there.

**Comment by rouazana** [comment]
See https://github.com/jmapio/jmap/pull/288

**Pull request closed without merging**